src/components/InventoryTable/TitleColumn.js

```diff
@@ -24,32 +24,35 @@ const onRowClick = (event, key, { loaded, onRowClick: rowClick, noDetail }) => {
 };
 
 /**
- * Helper function to generate first cell in plain inventory either with clickable detail or just data from attribut.
+ * Helper component to generate first cell in plain inventory either with clickable detail or just data from attribut.
  * This is later on used in redux in `renderFunc`.
- * @param {React.node} data React node with information that will be shown to user as column title.
+ * @param {React.node} children React node with information that will be shown to user as column title.
  * @param {string} id inventory UUID, used to navigate to correct URL.
  * @param {*} item row data, holds every information from redux store for currecnt row.
  * @param {*} props additional props passed from `EntityTable` - holds any props passed to inventory table.
  */
-const TitleColumn = (data, id, item, props) => (
+const TitleColumn = ({ children, id, item, ...props }) => (
   <div className="ins-composed-col sentry-mask data-hj-suppress">
-    <div key="os_release">{item?.os_release}</div>
+    {item?.os_release && <div key="os_release">{item?.os_release}</div>}
```

Comment:
I don't think that we are showing/using this anywhere, or?

Reply:
I'm not 100% sure what it even does. Would it provide another div in the title column with the os version along with the system name? If so, then I've never seen that.

Reply:
Yeah, it would do just that. I'll leave it in for now, but I think we can eventually remove it.